h2o-genmodel/src/main/java/hex/genmodel/AbstractMojoWriter.java

```diff
@@ -45,7 +45,7 @@ public AbstractMojoWriter(ModelDescriptor model) {
   /**
    * Override in subclasses to write the actual model data.
    */
-  protected abstract void writeModelData() throws IOException;
+  protected abstract void writeModelKV() throws IOException;
```

Comment:
renamed this as I needed to keep `writeModelData` for specific model implementations.
In `ModelMojoWriter`, `writeModelKV` is responsible to write both preprocessor related data + model specific data (= `writeModelData`).